.github/ISSUE_TEMPLATE/new_python_tool_pip.yml

```
@@ -0,0 +1,128 @@
+      placeholder: ex. js-deobfuscator
+    validations:
+      required: true
+  - type: input
+    id: tool_name
+    attributes:
+      label: Tool Name
+      description: |
+        The name of the tool being installed with `pip install <tool_name>`
+      placeholder: ex. magika
+    validations:
+      required: true
+  - type: dropdown
+    id: type
+    validations:
+      required: true
+    attributes:
+      label: Package type
+      description: |
+        - **`PIP`** - A Python tool installed with `pip install <tool_name>`.
+
+        For other types of tools, use a different issue template.
+      options:
+        - PIP
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: |
+        Tool's version number. Example: `1.4.11`. See the [_Version_ documentation in our wiki](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#version) for more details.
+      placeholder: ex. 1.4.11
+    validations:
+      required: true
+  - type: dropdown
+    id: category
+    validations:
+      required: true
+    attributes:
+      label: Category
+      description: Which category should this tool be installed to?
+      options:
+        - Command & Control
+        - Credential Access
+        - Debuggers
+        - Delphi
+        - Disassemblers
+        - dotNet
+        - Documents
+        - Exploitation
+        - File Information
+        - Forensic
+        - Hex Editors
+        - InnoSetup
+        - Java & Android
+        - Javascript
+        - Lateral Movement
+        - Memory
+        - Networking
+        - Packers
+        - Payload Development
+        - PE
+        - Persistence
+        - Privilege Escalation
+        - Productivity Tools
+        - Python
+        - Reconnaissance
+        - Registry
+        - Shellcode
+        - Utilities
+        - Visual Basic
+        - Web Application
+        - Wordlists
+  - type: input
+    id: authors
+    attributes:
+      label: Tool's authors
+      description: |
+        Comma separated list of the tool's authors. Example: `Elmo, Bert, Grover, Cookie Monster`.
+      placeholder: ex. Elmo, Bert, Grover, Cookie Monster
+    validations:
+      required: true
+  - type: input
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: Tool's description
+      description: |
+        The tool description should be short and not include version specific details or other information that is likely to change in a future version. Example: `FakeNet-NG is a dynamic network analysis tool.`
+  - type: input
+    id: dependencies
+    attributes:
+      label: Dependencies
+      description: |
+        The dependencies (tools/libraries that need to be installed) needed for installing or using this tool.
```

I think we should mention that Python3 is already required as it is a dependency for all pip tools 
```suggestion
        The dependencies (tools/libraries that need to be installed) needed for installing or using this tool. Do not include Python3, as it is already included by default.
```